docs/Routes.md

```diff
@@ -168,7 +171,24 @@ fastify.get('/', options, async function (request, reply) {
   reply.send(processed)
 })
 ```
-**Warning:** If you use `return` and `reply.send` at the same time, the first one that happens takes precedence, the second value will be discarded, a *warn* log will also be emitted if the value is not `undefined`.
+* **Warning:** If you use `return` and `reply.send` at the same time, the first one that happens takes precedence, the second value will be discarded, a *warn* log will also be emitted because you tried to send a response twice.
+* **Warning:** You can't return `undefined`. For more details read [promise-resolution](#promise-resolution).
```

Comment:
Why is this warning listed twice in the same section?

Reply:
Because there are two warnings it's just a matter of style.

Reply:
It's redundant.

Reply:
What's your suggestion? To headline it only once but keep a list ?

Reply:
![fastify](https://user-images.githubusercontent.com/321201/38933955-69b617f4-42e8-11e8-8727-9aaec2366f17.png)